controllers/replication.storage/volumereplication_controller.go

```diff
@@ -125,6 +125,8 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}
 	// remove the prefix keys in volume replication class parameters
 	parameters := filterPrefixedParameters(replicationParameterPrefix, vrcObj.Spec.Parameters)
+	schedulingTime := parameters["schedulingInterval"]
+	scheduleTime, _ := time.ParseDuration(schedulingTime)
```

Comment:
Whhy we are sending parameters?

Reply:
We want to update the last sync time after every interval and that is, scheduling interval and hence, it's required.

Reply:
from VolumeReplication CR schedulingInterval is optional